src/main/java/com/scaler/lld/design/assignments/builder/DatabaseConfigurationBuilder.java

```diff
@@ -1,6 +1,87 @@
 package com.scaler.lld.design.assignments.builder;
 
+import java.util.Objects;
+
 @WithBuilder
 public class DatabaseConfigurationBuilder {
 
-}
\ No newline at end of file
+    private String databaseUrl;
+    private String username ;
+    private String password ;
+    private int maxConnections;
+    private boolean enableCache;
+    private boolean isReadOnly;
+
+    private DatabaseConfigurationBuilder() {
+
+
+    }
+
+
+    //DatabaseConfigurationBuilderTest
+    @org.jetbrains.annotations.NotNull
+    @org.jetbrains.annotations.Contract(value = " -> new", pure = true)
+    public static DatabaseConfigurationBuilderTest builder() {
+        return new DatabaseConfigurationBuilderTest();
+    }
+    public static class DatabaseConfigurationBuilderTest{
+       private String databaseUrl;
+       private String username;
+       private String password;
+       private int maxConnections;
+       private boolean enableCache;
+       private boolean isReadOnly;
```

Comment:
Try to get rid of the field duplication here.

Reply:
Done the changes